### PR TITLE
Minor spelling fixes

### PR DESCRIPTION
**Pull request by modery**
Fixing some wrong spellings

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/teams-first-overview.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ To get started with your Teams First deployment you will need to meet at minimum
 4.  Organization's network prepared for Teams  
     To understand network preparation, read [Prepare your organization's network for Teams](prepare-network.md).
 
-5.  Allow network access to Exchange, Sharepoint, and OneDrive for Business in Microsoft 365 or Office 365: [Office 365 URLs and IP address ranges](/office365/enterprise/urls-and-ip-address-ranges).
+5.  Allow network access to Exchange, SharePoint, and OneDrive for Business in Microsoft 365 or Office 365: [Office 365 URLs and IP address ranges](/office365/enterprise/urls-and-ip-address-ranges).
```
Comment (JohanFreelancer9) [inline Teams/teams-first-overview.md:46]:
```suggestion
5.  Allow network access to Exchange, SharePoint, and OneDrive in Microsoft 365 or Office 365: [Office 365 URLs and IP address ranges](/office365/enterprise/urls-and-ip-address-ranges).
```

File: Teams/toc.yml [hunk 2]
```diff
@@ -306,7 +306,7 @@ items:
           href: location-of-data-in-teams.md
         - name: Microsoft Teams feedback privacy
           href: teams-privacy-feedback.md
-        - name: Block access to Sharepoint Online for specific users
+        - name: Block access to SharePoint Online for specific users
```
Comment (JohanFreelancer9) [inline Teams/toc.yml:309]:
```suggestion
        - name: Block access to SharePoint for specific users
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @modery, kindly commit the additional changes if you agree with it. Many thanks.

**Pull request merged**